### PR TITLE
OSDOCS-12932: 4.14.43 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-12932

Link to docs preview:
https://86435--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-43_release-notes

QE review:
N/A

Additional information:
Errata links will return 404 until errata are shipped live (12/18)

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3392,6 +3392,35 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+----
+$ oc adm release info 4.14.43 --pullspecs
+----
+
+[id="ocp-4-14-43-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a trailing period was not permitted in a Kubernetes object name per the DNS 1123 sub-domain name standard. If you configured your AWS DHCP option set with a custom domain name containing a trailing period, the logic for extracting the hostname of EC2 instances and turning it into Kubelet node names would not trim the hostnames for trailing periods. With this release, the logic is updated to trim trailing periods, and trailing periods are permitted within the domain name in the DHCP option set. (link:https://issues.redhat.com/browse/OCPBUGS-46057[*OCPBUGS-46057*])
+
+* Previously, {hcp}-based clusters were unable to authenticate through the `oc login` command. The web browser displayed an error when it attepted to retrieve the the token after selecting *Display Token*. with this release, `cloud.ibm.com` and other cloud-based endpoints are no longer proxied and authentication is successful. (link:https://issues.redhat.com/browse/OCPBUGS-44279[*OCPBUGS-44279*])
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:3417]:
```suggestion
* Previously, {hcp}-based clusters were unable to authenticate through the `oc login` command. The web browser displayed an error when it attepted to retrieve the the token after selecting *Display Token*. With this release, `cloud.ibm.com` and other cloud-based endpoints are no longer proxied and authentication is successful. (link:https://issues.redhat.com/browse/OCPBUGS-44279[*OCPBUGS-44279*])
```

**Review by bergerhoffer — COMMENTED** [review]
One typo, and then the advisory links don't work, but I'm assuming because the release isn't live yet

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (obrown1205) [reply release_notes/ocp-4-14-release-notes.adoc:3417]: thank you!

**Pull request merged**